test/inductor/test_torchinductor.py

```diff
@@ -1359,6 +1359,31 @@ def fn(a, b):
         )
         self.common(fn, (a, b), atol=1e-5, rtol=1e-5, check_lowp=False)
 
+    @skipCUDAIf(TEST_WITH_ROCM, "associative_scan is not supported on ROCm")
+    def test_custom_scan_op(self):
```

Comment:
Also add a test where associative_scan is called within a larger function that's compiled.